packages/code-du-travail-frontend/src/piwik.js

```diff
@@ -2,21 +2,20 @@ import ReactPiwik from "react-piwik";
 import { Router } from "../routes";
 import getConfig from "next/config";
 
-const onRouteChangeComplete = () => {
-  ReactPiwik.push(["trackPageView"]);
-};
-
 const {
   publicRuntimeConfig: { PIWIK_URL, PIWIK_SITE_ID }
 } = getConfig();
 
 if (typeof window !== "undefined" && PIWIK_URL && PIWIK_SITE_ID) {
-  new ReactPiwik({
+  const piwik = new ReactPiwik({
     url: PIWIK_URL,
     siteId: PIWIK_SITE_ID,
     trackErrors: true
   });
-  Router.events.on("routeChangeComplete", onRouteChangeComplete);
+  ReactPiwik.push(["trackPageView"]);
+  Router.events.on("routeChangeComplete", path => {
+    piwik.track({ path });
```

Comment:
path should not be a complete url `http://{hostname}/{pathname}` but a pathname since 
the Piwik.track will append `${window.location.origin}`
What do you think of using track(window.location) wich will fallback to (pathname + search)

Reply:
It turns out the path returned by the router is exactly pathname + search !

Reply:
the nextjs documentation is a bit misleading :(
> routeChangeComplete(url) - Fires when a route changed completely
> Here url is the URL shown in the browser. If you call Router.push(url, as) (or similar), then the value of url will be as.

https://github.com/zeit/next.js#router-events

Reply:
Plus we also have a package around the router, it might changes a few things too. I didn't read the doc, I simply `console.log` it to see the value.